probe_scraper/scrapers/git_scraper.py

```diff
@@ -41,6 +41,7 @@
     "mozilla-vpn-android": "2021-05-25 00:00:00",
     "rally-markup-fb-pixel-hunt": "2021-12-04 00:00:00",
     "rally-citp-search-engine-usage": "2022-04-15 00:00:00",
+    "relay-backend": "2024-05-09 00:00:00",
```

Comment:
This is needed because some of the earlier commits threw an error during a dry-run:
```
    Body: Error in processing commit f5beadd3e5552dfe29c57abd1b7a7318675e8840
    Body: Error in processing commit 69c650916cd2a2a4f04af6c98f1280e7a6c676ec
    Body: Error in processing commit 9f3c33cd21545a9151c55132afee8a981bb698c2
    Body: Error in processing commit 899f2fd59277dc38376de03000e1e51f95c3ae2c
    Body: Error in processing commit 700454a838a476c342cb0fd45b41f00eba0cfeb9
    Body: Error in processing commit ab2f13d258eba140aeb30af0714cac036cce6556
```